etc/concepts.tsv

```diff
@@ -0,0 +1,336 @@
+Newyoung	New (young)		
+Greedy	Greedy	2018	GREEDY
+Ikneel	I kneel	66	KNEEL
+Irise	I rise	3207	GO UP OR RISE (ONESELF)
+Icutatree	I cut a tree	3151	CUT (WITH AXE)
+Grandfather	Grandfather	1383	GRANDFATHER
+Icallaslave	I call a slave		
+Ipraytogodbeggod	I pray to god (beg god)	24	PRAY
+Grandmother	Grandmother	1496	GRANDMOTHER
+Son	Son	1620	SON
+DaughterMyThy	Daughter (My, Thy)	1357	DAUGHTER
+Toe	Toe	1389	TOE
+Rib	Rib	801	RIB
+Heel	Heel	980	HEEL
+Shoe	Shoe	1381	SHOE
+Trousers	Trousers	809	TROUSERS
+TownVillage	Town (Village)	3609	TOWN OR VILLAGE
+Village	Village	930	VILLAGE
+Market	Market	633	MARKET
+EarRing	Ear-Ring	770	EARRING
+Itch	Itch	148	ITCH
+SmallPox	Small-Pox	1054	SMALLPOX
+GuineaCornbearinglikeMaize	Guinea-Corn (bearing like Maize)		
+Mare	Mare	938	MARE
+Ivory	Ivory		
+Iflogachild	I flog a child		
+Ten	Ten	1515	TEN
+Eleven	Eleven	1706	ELEVEN
+Twelve	Twelve	1707	TWELVE
+Seven	Seven	1704	SEVEN
+Eight	Eight	1705	EIGHT
+Nine	Nine	1483	NINE
+Six	Six	1703	SIX
+Maize	Maize	506	MAIZE
+Twenty	Twenty	1710	TWENTY
+Idol	Idol	1945	IDOL
+Ink	Ink	2643	INK
+Onion	Onion	2366	ONION
+Thirteen	Thirteen	1708	THIRTEEN
+Fourteen	Fourteen	2451	FOURTEEN
+Fifteen	Fifteen	1709	FIFTEEN
+Sixteen	Sixteen	2454	SIXTEEN
+Seventeen	Seventeen	2453	SEVENTEEN
+Eighteen	Eighteen	1713	EIGHTEEN
+Nineteen	Nineteen	1714	NINETEEN
+Buttermelted	Butter (melted)		
+Camwood	Camwood		
+ChainFettersforneck	Chain (Fetters ?) (for neck)		
+Batlarge	Bat (large)		
+Pigeonwild	Pigeon (wild)		
+Yamwild	Yam (wild)		
+Rainwater	Rain (water)		
+cough	cough		
+Spoonofwood	Spoon (of wood)		
+Fingerthumb	Finger (thumb)	1781	THUMB
+Toethebigone	Toe (the big one)		
+crossbow	cross-bow	3193	CROSSBOW
+Pigeondomestic	Pigeon (domestic)		
+body	body	1480	BODY
+ArmletorBraceletofbrass	Armlet or Bracelet (of brass)		
+Firewooddrywood	Firewood (dry wood)		
+young	young	1207	YOUNG
+Goldbrass	Gold (brass)		
+Cassadawild	Cassada (wild)		
+Veinrope	Vein (rope)		
+Needleforornament	Needle (for ornament)		
+Oldofpersons	Old (of persons)	2112	OLD (AGED)
+Oldofthings	Old (of things)	2113	OLD (USED)
+Ieatyam	I eat yam		
+witch	witch	824	WITCH
+silver	silver	759	SILVER
+Sheep	Sheep	1331	SHEEP
+myyoungerson	my younger son		
+Isew	I sew	1457	SEW
+Milkofcows	Milk (of cows)		
+Milkofwomen	Milk (of women)	2129	BREAST OR MILK
+NailofFinger	Nail (of Finger)	1258	FINGERNAIL
+Oldofmen	Old (of men)	565	OLD (OF MAN)
+Palmtreeayoungone	Palm-tree (a young one)		
+Ieatyams	I eat yams		
+Ieatcorn	I eat corn		
+BenchChairformen	Bench, Chair (for men)		
+BenchChairforwomen	Bench, Chair (for women)		
+awalledtown	a walled town		
+Doorofgrass	Door (of grass)		
+Doorofwood	Door (of wood)		
+Milkfresh	Milk (fresh)		
+Milksour	Milk (sour)		
+Rootstump	Root (stump)	241	TREE STUMP
+Pigeontame	Pigeon (tame)		
+Lizardthecommononemale	Lizard (the common one) (male)		
+ThelargeredheadedLizardfemale	The large red-headed Lizard (female)		
+Leafgrass	Leaf (grass)		
+New	New	1231	NEW
+Oldpersons	Old (persons)	566	OLD (OF WOMAN)
```

Comment:
I would also unmap this one. There is no indication that the concept refers exclusively to women.

Reply:
Was a mistake